lib/tool_shed/galaxy_install/tool_dependencies/recipe/step_handler.py

```diff
@@ -845,19 +845,26 @@ def execute_step( self, tool_dependency, package_name, actions, action_dict, fil
                                    destination_dir=os.path.join( action_dict[ 'destination_directory' ] ) )
         return tool_dependency, None, None
 
-    def move_directory_files( self, current_dir, source_dir, destination_dir ):
-        source_directory = os.path.abspath( os.path.join( current_dir, source_dir ) )
-        destination_directory = os.path.abspath(os.path.join(destination_dir))
+    def move_directory_files(self, current_dir, source_dir, destination_dir):
+        source_directory = os.path.abspath(os.path.join(current_dir, source_dir))
+        destination_directory = os.path.join(destination_dir)
```

Comment:
We may need this to be
`destination_directory = os.path.abspath(os.path.join(destination_dir))`